canonical-web-and-design/snapcraft.io
# Migrate to Vanilla 2.3.0

For demo purposes of Vanilla 2.0 migration

Issues that need resolving/ releasing by Vanilla before this can be merged to master:
- [x] [Spacing between paragraphs in rows in `p-card` ](https://github.com/canonical-web-and-design/vanilla-framework/issues/2447) [CLOSED]
- [x] [`p-stepped-list` number becomes an oval if the title wraps](https://github.com/canonical-web-and-design/vanilla-framework/issues/2424) [CLOSED]
- [x] custom grid styles (like generated banner) break when used inside vanilla grid (https://github.com/canonical-web-and-design/vanilla-framework/issues/2444) [CLOSED]
- [x] ensure metrics tooltips are positioned correctly on dashboard and metrics page

Non blocking, but nice to have:
- [ ] visited pagination buttons get blue (canonical-web-and-design/vanilla-framework#2458)
- [ ] pagination is not exactly centred (canonical-web-and-design/vanilla-framework#2459)
- [ ] misaligned checkboxes (https://github.com/canonical-web-and-design/vanilla-framework/issues/2461)

### QA

Most of the changes were already reviewed and QA'ed in the individual PRs.
This QA is meant to make a final check if noting is significantly broken.

- go to https://snapcraft-io-canonical-web-and-design-pr-2144.run.demo.haus/
- click on things
- visit store, search, visit some snap details page, visit distro pages
- have a look at first snap flow
- sign in
- check some snaps publisher pages (listing, metrics, release)
- try some obscure features (report snap modal, embedded cards, etc)
